examples/1d/parameters.toml

```
@@ -0,0 +1,47 @@
+####################################################################################################
+# Simulation
+ndims = 1
+# equations = "CompressibleEulerEquations"
+equations = "LinearScalarAdvectionEquation"
+advectionvelocity = [1.0] # Need only for linarscalaradvection
+
+initial_conditions = "initial_conditions_convergence_test"
+#initial_conditions = "initial_conditions_constant"
+#initial_conditions = "initial_conditions_linear_x_y"
+surface_flux = "flux_lax_friedrichs"
+# source_terms =
+t_start = 0.0
+t_end   = 1.0
+
+# restart = true
+# restart_filename = "out/restart_000100.h5"
+
+
+####################################################################################################
+# Solver
+solver = "dg"
+polydeg = 4
```

```suggestion
polydeg = 3
```

To be consistent with 2D/3D.